.gitignore

```diff
@@ -21,6 +21,8 @@ test.out
 # Daemon executables & libraries
 daemons/compute/client-c/client
 daemons/compute/client-c/client.a
+daemons/compute/client-go/client-go
+daemons/compute/server/nnf-dm
```

Comment:
These lines for client-go/client-go and server/nnf-dm are already in master.

Reply:
Thanks for pointing that out. I'll rebase and remove my additions.